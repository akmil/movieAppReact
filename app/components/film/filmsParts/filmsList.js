import React from 'react';
// import {ResultsBody} from './../../resultsBox/resultsBox';
import VisibleMovieList from './../container-redux/VisibleMovieList';

export const FilmsList = () => (
    <VisibleMovieList></VisibleMovieList>
);
