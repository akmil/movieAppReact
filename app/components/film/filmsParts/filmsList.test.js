import React from 'react';
// import { shallow } from 'enzyme';
import {FilmsList} from './filmsList';
import renderer from 'react-test-renderer';

describe('<films FilmsList />', () => {
    it('should render SearchButton without errors', () => {
        const wrapper = renderer.create(<FilmsList/>);
        expect(wrapper).toMatchSnapshot();
    })
});