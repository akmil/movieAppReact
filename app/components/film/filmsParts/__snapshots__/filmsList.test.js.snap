// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<films FilmsList /> should render SearchButton without errors 1`] = `
<div
  className="results-list-box"
>
  <ul
    className="results-list"
  >
    <li>
      <img
        alt="Movie title 4"
        src="https://images.pexels.com/photos/460307/pexels-photo-460307.jpeg?auto=compress&cs=tinysrgb&h=350"
      />
      <div
        className="col-50"
      >
        <p
          className="title"
        >
          Movie title 4
        </p>
        <p
          className="genre"
        >
          Drama2
        </p>
      </div>
      <div
        className="col-50 text-align-right"
      >
        <span
          className="release-date"
        >
          1299
        </span>
      </div>
    </li>
    <li>
      <img
        alt="Movie title 5"
        src="https://images.pexels.com/photos/460307/pexels-photo-460307.jpeg?auto=compress&cs=tinysrgb&h=350"
      />
      <div
        className="col-50"
      >
        <p
          className="title"
        >
          Movie title 5
        </p>
        <p
          className="genre"
        >
          Drama2
        </p>
      </div>
      <div
        className="col-50 text-align-right"
      >
        <span
          className="release-date"
        >
          1395
        </span>
      </div>
    </li>
    <li>
      <img
        alt="Movie title 1"
        src="https://images.pexels.com/photos/460307/pexels-photo-460307.jpeg?auto=compress&cs=tinysrgb&h=350"
      />
      <div
        className="col-50"
      >
        <p
          className="title"
        >
          Movie title 1
        </p>
        <p
          className="genre"
        >
          Drama
        </p>
      </div>
      <div
        className="col-50 text-align-right"
      >
        <span
          className="release-date"
        >
          1995
        </span>
      </div>
    </li>
    <li>
      <img
        alt="Movie title 2"
        src="https://images.pexels.com/photos/257092/pexels-photo-257092.jpeg?auto=compress&cs=tinysrgb&h=350"
      />
      <div
        className="col-50"
      >
        <p
          className="title"
        >
          Movie title 2
        </p>
        <p
          className="genre"
        >
          Comedy
        </p>
      </div>
      <div
        className="col-50 text-align-right"
      >
        <span
          className="release-date"
        >
          1998
        </span>
      </div>
    </li>
    <li>
      <img
        alt="Movie title 3"
        src="https://images.pexels.com/photos/374633/pexels-photo-374633.jpeg?auto=compress&cs=tinysrgb&h=350"
      />
      <div
        className="col-50"
      >
        <p
          className="title"
        >
          Movie title 3
        </p>
        <p
          className="genre"
        >
          Drama
        </p>
      </div>
      <div
        className="col-50 text-align-right"
      >
        <span
          className="release-date"
        >
          1991
        </span>
      </div>
    </li>
  </ul>
</div>
`;
